src/Worker/DBClean.php

```diff
@@ -323,10 +323,11 @@ private static function removeOrphans($stage) {
 			Config::set('system', 'dbclean-last-id-9', $last_id);
 		} elseif ($stage == 10) {
 			$last_id = Config::get('system', 'dbclean-last-id-10', 0);
+			$days = intval(Config::get('system', 'dbclean_expire_conversation', 7));
 
 			logger("Deleting old conversations. Last created: ".$last_id);
 			$r = dba::p("SELECT `received`, `item-uri` FROM `conversation`
-					WHERE `received` < UTC_TIMESTAMP() - INTERVAL 90 DAY
+					WHERE `received` < UTC_TIMESTAMP() - INTERVAL $days DAY
```

Comment:
Please use ```?``` instead of ```$days```. See some lines above in the code:
```WHERE `thread`.`received` < UTC_TIMESTAMP() - INTERVAL ? DAY```

The ```?``` is a placeholder that you then pass as parameter in that command.